src/sentry/tasks/derive_code_mappings.py

```diff
@@ -69,62 +53,30 @@ def derive_code_mappings(organization_id: int, dry_run=False) -> None:
 
     trees: JSONData = installation.get_trees_for_org()
```

Comment:
This is the slow part of the request.
I will make [WOR-2256](https://getsentry.atlassian.net/browse/WOR-2256) a priority.

Reply:
@snigdhas Do you know if we will make the request for one event per org per hour? Or can we have N events for the same org within an hour?

Reply:
We should only have one event per org, because of the cache mechanism introduced in [#40882](https://github.com/getsentry/sentry/pull/40882/files#diff-b8b58ee6be5cf471f56bfbcf8d91a64bdfca7ae9800245997d6eb5acfff13aceR611)